tests/foreman/ui/test_repository.py

```diff
@@ -339,6 +338,24 @@ def test_positive_discover_module_stream_repo_via_existing_product(session, modu
 
     :BZ: 1676642
     """
+    repo_name = gen_string('alpha')
+    repo_label = gen_string('alpha')
+    product = entities.Product(organization=module_org).create()
+    with session:
+        session.organization.select(org_name=module_org.name)
```

Comment:
@Gauravtalreja1 you don't need to explicitly select organization.
Can you re-test this on 6.8 and let me know if it's still working fine for you?
I was not able to discover repos, probably something wrong with my satellite.
Otherwise it LGTM. :+1:

Reply:
@jameerpathan111 Selecting Org is used across the almost whole test_repository, would you like me to update this everywhere?

Sorry I have issues with my robottelo env for running UI tests, so using PRT for test results, and I updated this PR, request to re-review.

Reply:
that's fine, merging the pr. Somehow I missed comments from reviewers.